daemon/graphdriver/devmapper/README.md

```diff
@@ -216,3 +216,26 @@ Here is the list of supported options:
     Example use:
 
     ``docker -d --storage-opt dm.blkdiscard=false``
+
```

Comment:
There were some awkward constructions that made this hard to parse and obscured the behavior.  The Notes and Warning on top of each other indicate structural issues. Hopefully, I cleared this up with some input from @tiborvass  (ty!).

Markdown: https://gist.github.com/moxiegirl/ade0634692f7423bf474

---->

*  `dm.override_udev_sync_check`

Overrides the kernel's `udev` synchronization checks. `udev` is the device
manager for the Linux kernel. By default, this value is `false` and the kernel
does not support override `udev` synchronization checks.  

Under the `false` setting, race conditions occur between the`devicemapper` and `udev` during create and cleanup. The race conditions result in errors and the
`devicemapper` fails. (For information on these failures, see
[docker#4036](https://github.com/docker/docker/issues/4036))

To allow the `docker` daemon to function properly with the `devicemapper`
driver, set `dm.override_udev_sync_check` to true:  

	$ docker -d --storage-opt dm.override_udev_sync_check=true
	
When this value is `true`, the  `devicemapper` continues and simply warns you the errors are happening 

**Note**: The ideal is to pursue a `docker` daemon and environment that does
support synchronizing with Udev. For further discussion on this topic, see 
[docker#4036](https://github.com/docker/docker/issues/4036).  Otherwise, set this flag for migrating existing daemons to a daemon
with a supported environment.